add-domain

```diff
@@ -522,3 +522,11 @@ pdfkinghub.xyz
 tracebb.trackingsystem.xyz
 postgo.webpostgo.icu
 totaltrkonups2298.fitritomoastramencostea.com
+tracktt.deliveryoffer.top
+mail-e9eredelivery.com
+trackpp.searchmyitem.xyz
+steamcomnnulty.com
+yahoofbuyv.weebly.com
+attonlineservicexx.weebly.com
+julyemailclosurer.weebly.com
+support-1oakyf-league-of-legends.000webhostapp.com
```

Comment:
Active:

![image](https://user-images.githubusercontent.com/44526987/126351255-cd920897-b576-458c-aaeb-13b047f916f9.png)


You can test this with the special rules for @PyFunceble: 

https://github.com/funilrys/PyFunceble/blob/89f2b69c4f02657124d3a05d6b73e43ca5a46c06/PyFunceble/checker/availability/extra_rules.py#L84-L86